mibitracker/request_helpers.py

```diff
@@ -500,6 +501,26 @@ def get_mibi_image(self, image_id):
         tiff_data = self.download_file(tiff_path)
         return tiff.read(tiff_data)
 
+    def get_single_channel(self, image_id, channel_name):
```

Comment:
what do you think of `get_channel_data`?

Reply:
Yes.